measures/2019/measures-data.json

```diff
@@ -17303,14 +17303,13 @@
     "isInverse": false,
     "isIcdImpacted": false,
     "isToppedOutByProgram": false,
-    "isClinicalGuidelineChanged": false,
+    "isClinicalGuidelineChanged": true,
     "category": "quality",
     "isRegistryMeasure": false,
     "isRiskAdjusted": false,
     "submissionMethods": [
       "claims",
       "electronicHealthRecord",
-      "cmsWebInterface",
```

Comment:
Do not remove `cmsWebInterface` as a valid submission method. WI _will_ still submit this measure, but it will get the denominator reduction for `QUALITY-065`